SR2020/2020-05-22-virtual-competition-transition.md

```diff
@@ -0,0 +1,26 @@
+---
+to: sr2020-teams
+subject: Transitioning Student Robotics 2020 into a virtual competition
```

Comment:
I'm concerned at the length of the subject here, might be cut off.

As the sender will show as us, I'd be tempted to redact that
```suggestion
subject: Transitioning to a virtual competition
```